core/controllers/editor.py

```diff
@@ -271,11 +263,7 @@ def put(self, exploration_id):
 
         elif make_community_owned:
             exploration = exp_services.get_exploration_by_id(exploration_id)
-            try:
-                exploration.validate(strict=True)
-            except utils.ValidationError as e:
```

Comment:
In the line above, we are using `exploration.validate(strict=True)`. Strict means that it will raise an exception from the validate function and hence this line becomes unreachable.

Reply:
Why is it unreachable? The whole point of try/except is to catch an exception, and if a ValidationError is raised then the except clause will be triggered.

Reply:
Done